includes/class-wcs-importer.php

```diff
@@ -143,7 +144,7 @@ public static function import_subscription( $data ) {
 		global $wpdb;
 
 		self::$row  = $data;
-		$set_manual = false;
+		$set_manual = $requires_manual_renewal = false;
```

Comment:
have not seen this, did know its kinda possible. http://stackoverflow.com/questions/11651594/assign-same-value-to-multiple-variables